Dockerfile

```diff
@@ -0,0 +1,15 @@
+# ---- Base image - order important ----
+FROM hmcts/ccd-definition-processor:latest as base
+
+# ----        Runtime image         ----
+FROM hmcts/ccd-definition-importer:latest as runtime
+
+COPY package.json yarn.lock ./
+COPY /definitions/divorce/xlsx /
+ADD ./config "/config"
+RUN apk add --no-cache nodejs yarn
```

Comment:
do we need to install a specific version of nodejs? In the package.json, the node is declared as `>=8.11.0 <9.0.0`

Reply:
@hosainnet any thoughts above ^

Reply:
@charlesluokb not really - this app is just to pass the pipeline healthcheck (providing a `/health` endpoint etc)

Reply:
thanks 👍